proto/src/determined/api/v1/job.proto

```diff
@@ -52,6 +54,8 @@ message GetJobsV2Request {
   OrderBy order_by = 3;
   // Filter to jobs with states among those given.
   repeated determined.job.v1.State states = 4;
+  // The resource manager for the request.
+  string resource_manager = 5;
```

Comment:
I think we will also need to update the move job resource pool proto definition? 

https://github.com/determined-ai/determined/blob/bccdf0c44ed864646c19aa1d2b071eb25c1fcb9e/proto/src/determined/job/v1/job.proto#L173C1-L173C30